Build error in linux

When I try to build redream in Ubuntu 16.10, I have an error : 

[ 86%] Linking CXX executable redream
deps/sdl2-2.0.4/libSDL2.a(SDL_waylandvideo.c.o) : Dans la fonction « wl_registry_bind » :
/usr/include/wayland-client-protocol.h:1061 : référence indéfinie vers « wl_proxy_marshal_constructor_versioned »
/usr/include/wayland-client-protocol.h:1061 : référence indéfinie vers « wl_proxy_marshal_constructor_versioned »
/usr/include/wayland-client-protocol.h:1061 : référence indéfinie vers « wl_proxy_marshal_constructor_versioned »
/usr/include/wayland-client-protocol.h:1061 : référence indéfinie vers « wl_proxy_marshal_constructor_versioned »
/usr/include/wayland-client-protocol.h:1061 : référence indéfinie vers « wl_proxy_marshal_constructor_versioned »
deps/sdl2-2.0.4/libSDL2.a(SDL_waylandvideo.c.o):/usr/include/wayland-client-protocol.h:1061 : encore plus de références indéfinies suivent vers « wl_proxy_marshal_constructor_versioned »
deps/libsoundio/libsoundio.a(jack.c.o) : Dans la fonction « soundio_jack_init » :
jack.c:(.text+0x19eb) : référence indéfinie vers « jack_set_port_rename_callback »
collect2: error: ld returned 1 exit status
CMakeFiles/redream.dir/build.make:2006 : la recette pour la cible « redream » a échouée
make[2]: *** [redream] Erreur 1
CMakeFiles/Makefile2:109 : la recette pour la cible « CMakeFiles/redream.dir/all » a échouée
make[1]: *** [CMakeFiles/redream.dir/all] Erreur 2
Makefile:127 : la recette pour la cible « all » a échouée
make: *** [all] Erreur 2

--- a/deps/sdl2-2.0.4/src/video/wayland/SDL_waylanddyn.h
+++ b/deps/sdl2-2.0.4/src/video/wayland/SDL_waylanddyn.h
@@ -79,6 +79,7 @@ void SDL_WAYLAND_UnloadSymbols(void);
 #define wl_proxy_get_user_data (*WAYLAND_wl_proxy_get_user_data)
 #define wl_proxy_add_listener (*WAYLAND_wl_proxy_add_listener)
 #define wl_proxy_marshal_constructor (*WAYLAND_wl_proxy_marshal_constructor)
+#define wl_proxy_marshal_constructor_versioned (*WAYLAND_wl_proxy_marshal_constructor_versioned)
 
 #define wl_seat_interface (*WAYLAND_wl_seat_interface)
 #define wl_surface_interface (*WAYLAND_wl_surface_interface)

--- a/deps/sdl2-2.0.4/src/video/wayland/SDL_waylandsym.h
+++ b/deps/sdl2-2.0.4/src/video/wayland/SDL_waylandsym.h
@@ -55,6 +55,9 @@ SDL_WAYLAND_SYM(void, wl_list_insert_list, (struct wl_list *, struct wl_list *))
 SDL_WAYLAND_MODULE(WAYLAND_CLIENT_1_4)
 SDL_WAYLAND_SYM(struct wl_proxy *, wl_proxy_marshal_constructor, (struct wl_proxy *, uint32_t opcode, const struct wl_interface *interface, ...))
 
+SDL_WAYLAND_MODULE(WAYLAND_CLIENT_1_10)
+SDL_WAYLAND_SYM(struct wl_proxy *, wl_proxy_marshal_constructor_versioned, (struct wl_proxy *proxy, uint32_t opcode, const struct wl_interface *interface, uint32_t version, ...))
+
 SDL_WAYLAND_INTERFACE(wl_seat_interface)
 SDL_WAYLAND_INTERFACE(wl_surface_interface)
 SDL_WAYLAND_INTERFACE(wl_shm_pool_interface)

